Core/src/com/infiniteautomation/mango/jwt/JwtSignerVerifier.java

```diff
@@ -47,36 +48,39 @@ public abstract class JwtSignerVerifier<T> {
     public static final String INCORRECT_TYPE_EXPECTED_CLAIM_MESSAGE_TEMPLATE = "Expected %s claim to be of type: %s, but was: %s.";
     public static final String MISSING_TYPE_EXPECTED_CLAIM_MESSAGE_TEMPLATE = "Expected %s claim to be of type: %s, but was not present.";
 
+    private final Clock clock;
     private KeyPair keyPair;
     private JwtParser parser;
 
     protected final Logger log;
 
-    protected JwtSignerVerifier() {
+    protected JwtSignerVerifier(Clock clock) {
+        this.clock = clock;
```

Comment:
Modified API in a patch release, might be safer to not backport the tests (this change was for testing only).

Reply:
Ok, undoing it.